test/dialect/mssql/test_reflection.py

```diff
@@ -319,6 +319,30 @@ def test_indexes_cols_with_spaces(self):
 
         eq_(set(list(t2.indexes)[0].columns), set([t2.c["x col"], t2.c.y]))
 
+    @testing.provide_metadata
+    def test_indexes_with_filtered(self):
+        metadata = self.metadata
+
+        t1 = Table(
+            "t",
+            metadata,
+            Column("id", Integer, primary_key=True),
+            Column("x", types.String(20)),
+            Column("y", types.Integer),
+        )
+        Index("idx_x", t1.c.x, mssql_where=t1.c.x == "test")
+        Index("idx_y", t1.c.y, mssql_where=t1.c.y >= 5)
+        metadata.create_all()
```

Comment:
**mike bayer** (zzzeek) wrote:

metadata.create_all(connection)

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/2142

Reply:
**Ramon Williams** (RamonWill) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/2142